applications/asset_tracker_v2/src/ext_sensors/Kconfig

```
@@ -9,6 +9,29 @@ menuconfig EXTERNAL_SENSORS
 
 if EXTERNAL_SENSORS
 
+config EXTERNAL_SENSORS_IMPACT_DETECTION
+	bool "Use impact detection feature"
```

Please remove `use` as per guide lines:                                                                                                                                                                                                                                                                                                                                   
https://docs.zephyrproject.org/latest/build/kconfig/tips.html#prompt-strings                                                                                                                                                                                                                                                                                              
> For a Kconfig symbol that enables a driver/subsystem FOO, consider having just “Foo” as the prompt, instead of “Enable Foo support” or the like.

Also, isn't everything you can configure in Kconfig to be considered `a feature` ?

I suggest reducing to:
```suggestion
	bool "Impact detection"
```